examples/pytorch/rgcn/experimental/README.md

```diff
@@ -88,6 +88,58 @@ We can get the performance score at the second epoch:
 Val Acc 0.4323, Test Acc 0.4255, time: 128.0379
 ```
 
+## Partition a graph with ParMETIS
+
+It has four steps to partition a graph with ParMETIS for DGL's distributed training.
+More details about the four steps are explained in our
+[user guide](https://doc.dgl.ai/guide/distributed-preprocessing.html).
+
+### Step 1: write the graph into files.
+
+The graph structure should be written as a node file and an edge file. The node features and edge features
+can be written as DGL tensors. `write_mag.py` shows an example of writing the OGB MAG graph into files.
+
+```bash
+python3 write_mag.py
```

Comment:
Run the command will get an error:

Traceback (most recent call last):
  File "write_mag.py", line 42, in <module>
    node_data = th.stack(node_data, 1)
RuntimeError: Expected object of scalar type long int but got scalar type bool for sequence element 1.